app/views/tasks/_form.html.slim

```diff
@@ -40,10 +40,10 @@
             = f.text_area :internal_description, class: 'form-control'
           .field-element
             = f.label :uuid, Task.human_attribute_name('uuid'), class: 'form-label'
-            = f.text_field :uuid, class: 'form-control'
+            = f.text_field :uuid, type: :uuid, class: 'form-control not-allowed', disabled: true, title: t('.unchangeable_uuid'), 'data-bs-toggle': 'tooltip', pattern: '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$'
           .field-element
             = f.label :parent_uuid, Task.human_attribute_name('parent_uuid'), class: 'form-label'
-            = f.text_field :parent_uuid, class: 'form-control'
+            = f.text_field :parent_uuid, type: :uuid, class: 'form-control', pattern: '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$'
```

Comment:
You could extract the pattern to a useful constant / class method.

Reply:
I went with a method in the `ApplicationHelper` for now. There, we already have minor helpers for views, so that it aligns with the remaining architecture.

Reply:
As discussed in person, we agreed that the newly-introduced method is not ideal (since it is just returning a static string), but found it to be the best-fitting solution in the current architecture.